spec/helper.js

```diff
@@ -64,7 +85,9 @@ helper.startServer = function (overrides, endpointOpt) {
 };
 
 helper.stopServer = function () {
-    return helper.injector.get('app').stop();
+    if (helper.injector) {
```

Comment:
We could imagine that some unhandled exceptions are thrown before https://github.com/RackHD/on-core/blob/master/spec/helper.js#L163, then mocha will go on with the `after` section, then helper.injector would be undefined.